app/uk/gov/hmrc/gform/gform/SectionRenderingService.scala

```diff
@@ -501,11 +500,13 @@ class SectionRenderingService(frontendAppConfig: FrontendAppConfig) {
     hmrcTP: HmrcTaxPeriod) = {
 
     val taxPeriodList = obligations match {
-      case RetrievedObligations(listOfObligations) => listOfObligations.toList
+      case RetrievedObligations(listOfObligations) => listOfObligations
       case _                                       => List[TaxPeriodInformation]()
     }
 
+    import cats.syntax.eq._
```

Comment:
Is this import really needed? At the top of this file we have `import cats.implicits._`

Reply:
Lance and I have found that for some reason if we don't have this import here then we can't use type-safe equality a few lines below